examples/v2beta1/deepspeed/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Official PyTorch image with CUDA support
+FROM pytorch/pytorch:1.9.0-cuda11.1-cudnn8-runtime
```

Comment:
Haven't tried yet, it could be better to use this `mpioperator/base` image and just installing CUDA dependencies for DS with additional PyTorch / Tensorflow configurations.

Reply:
Actually, probably better to use `mpioperator/openmpi`. If you can make it work, that'd be great, as proof that the base images can be extended. I couldn't get tensorflow to work.

Reply:
Will try to make it work for both and patch to the PR 👍